README.md

```diff
@@ -26,3 +26,13 @@ Alex Redfish. [Check their art](https://www.artstation.com/artwork/5wm5W), it's
 
 - [Poussinou](https://github.com/Poussinou) - F-Droid and Google Play Store links
 - Mister klaid - Russian translations
+
+## Example
+#### Actual GIF
+
+<iframe src="https://giphy.com/embed/lNG8ZbSxsj9QcNikGJ" width="240" height="240" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/lNG8ZbSxsj9QcNikGJ">via GIPHY</a></p>
+
+#### Set as Wallpaper
+[<img src="homescreen.jpeg"
+     alt="Man hitting walls"
+     height="400">](https://gph.is/g/ZWdw6rL)
```

Comment:
If we would showcase a screenshot, then I would have that as part of the repo, and not rely on giphy as the source, in case they delete the gif at some point

Reply:
> Hey there! Thanks for the contribution.
> 
> Adding an exemple is a nice idea, but we shouldn't simply take the work of an artist like that. Looking at her website (https://www.boglio.com/gifs), she creates some cool gif, but I don't think they are creative common, so I don't feel at peace with just showing her gifs without asking first. She does have some cool images though.

I agree with you. Had some doubts regarding the license of the Gifs, so embedded them instead. I checked Gipfy.com's Commuity Guidelines and noticed that it mentions supporting the creators by embedding using their share tools.
Hence the embed.

Reply:
I had some intial doubts about the licensing, so thought of embedding the gif.
Regarding this:


> Hey there! Thanks for the contribution.
> 
> Adding an exemple is a nice idea, but we shouldn't simply take the work of an artist like that. Looking at her website (https://www.boglio.com/gifs), she creates some cool gif, but I don't think they are creative common, so I don't feel at peace with just showing her gifs without asking first. She does have some cool images though.

I'll look for some creative commons licensed gifs, and make the necessary changes.